Feature/gql ws link (#106)
gql-dart/gql: gql_websocket_link/CHANGELOG.md

> ## 0.2.0

Reviewer: 0.1.0 was never published as stable, so you can keep it 0.1.0 without bumping

> ## 0.1.1
> 
> The custom client has been removed.
> The library is now using WebSocketChannel as a base for the connection. 
> 
> ## 0.1.0
> 
> - Websocket link

Reviewer: ```suggestion
## 0.1.0

- port Websocket link from `graphq-flutter`
- use `WebSocketChannel` as a base for the connection
```
